src/common/header.rs

```diff
@@ -47,6 +57,14 @@ macro_rules! standard_headers {
     }
 }
 
+impl From<&str> for Header {
+    /// Gets a header from the given string representing the header name.
+    fn from(value: &str) -> Header {
+        Header::from(value.to_string())
+    }
+}
+
+
 standard_headers! {
```

Comment:
Do you think it might be better to read these from a file in the future or have you already hard coded everything?

Reply:
Oh maybe. We wouldn't be able to define constants for headers that way though. Might also be better performance-wise to have them hard coded.

Reply:
Ok. I was just wondering in case we would ever need to add more.